source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+  font-size: 1rem; // 16px
+
+  hx-def {
+    display: flex;
+
+    hx-dt {
+      #grid.span(2);
+      color: @gray-750;
+      display: block;
+      flex-shrink: 1;
+      font-weight: 400;
+      text-align: right;
+    }
+
+    hx-dd {
+      #grid.span(10);
+      color: @gray-900;
+      display: block;
+      flex-shrink: 1;
+      margin-left: 1.25rem; // 20px
+      word-break: break-all;
```

Comment:
I know I suggested this, but we probably need to make this property a helper class. Please extract to a helper class.